AFQ/api.py

```diff
@@ -94,7 +100,20 @@ def make_bundle_dict(bundle_names=BUNDLES, seg_algo="afq", resample_to=False):
         If set, templates will be resampled to the affine and shape of this
         image.
     """
+    logger = logging.getLogger('AFQ.api')
     if seg_algo == "afq":
+        if "FP" in bundle_names and "Occipital" in bundle_names:
+            logger.warning((
+                f"FP and Occipital bundles are co-located, and AFQ"
+                f" assigns each streamline to only one bundle."
+                f" Only Occipital will be used."))
+            bundle_names.remove("FP")
+        if "FA" in bundle_names and "Orbital" in bundle_names:
+            logger.warning((
+                f"FA and Orbital bundles are co-located, and AFQ"
+                f" assigns each streamline to only one bundle."
+                f" Only Orbital will be used."))
+            bundle_names.remove("FA")
```

Comment:
I think we also need:

```suggestion
            bundle_names.remove("FA")
            if "FA" in bundle_names and "AntFrontal" in bundle_names:
            logger.warning((
                f"FA and AntFrontal bundles are co-located, and AFQ"
                f" assigns each streamline to only one bundle."
                f" Only AntFrontal will be used."))
            bundle_names.remove("FA")
            
```